Readme.md

```diff
@@ -70,20 +66,31 @@ authenticate users. This file is uploaded to AWS secrets manager using `make set
 - Populate `FUS_ADMIN_EMAILS` with a list of admins to assigned upon creating the fusillade deployment. This
   is only used when fusillade is first deployed to create the first users. Afterwards this variable has no effect. If
   more admins are required assign a user the admin role.
-- Environment variables can be set in `environment.local` for convenience.
+- Environment variables can be set in `./deployment/${FUS_DEPLOYMENT_STAGE}/environment.local` for each 
+  deployment for convenience.
 - **Optionally** Before deploying fusillade you can modify the [default policies and roles](../blob/master/policies) 
  to suite your needs. The `default_admin_role.json` is policy attached to the fusillade_admin role created during 
  deployment. The `default_group_policy.json` is assigned to all new group when they are created. The 
  `default_user_role.json` is the role assigned to the group `default_user` which is created during deployment. All of 
  these policies and role can be modified after deployment using the fusillade API.
 
-## Set secrets
+## Set Secrets
```

Comment:
`./scripts/set_secrets`

Reply:
is this the same set_secrets as the dss? can you throw an example of the `echo | ./scripts/set_secret.py --secret-name` here?

Reply:
added an example commands